plugins/rtmp-services/data/schema/service-schema-v5.json

```diff
@@ -198,6 +198,14 @@
                         "$ref": "#/definitions/saneUrl",
                         "description": "Link that provides additional info about the service, presented in the UI as a button next to the services dropdown."
                     },
+                    "ertmp_configuration_url": {
```

Comment:
As this is not part of the eRTMP spec, it'd be worth naming this something a little more generic. For example, `custom_encoding_configuration_url`.

Reply:
Is this endpoint meant to be exclusively used for multitrack video? Or would it be reasonable for this to be something like `ingest_configuration_url` or `recommended_configuration_url`

Reply:
how about `multitrack_video_configuration_url`? this is only required for enhanced broadcasting/multitrack video, though the mechanism doesn't enforce that multiple video tracks are being used or similar

Reply:
I would like to personally suggest just `track_configuration_url`. I could foresee audio tracks being handled through this URL in the future too.

Reply:
Since all of the code work has been focused on multitrack video, I am fine with `multitrack_video_configuration_url` unless someone can justify why it should be more generic. If it should/must be more generic, I could understand `track_configuration_url` or `multitrack_configuration_url`.

Reply:
renamed things to `multitrack_video(_configuration_url)` in https://github.com/obsproject/obs-studio/pull/10543#commits-pushed-ae696c2

Reply:
Resolving this thread for now. If there are still concerns about the naming, please feel free to comment or open a new feedback thread.